M2/Macaulay2/packages/Complexes/ChainComplexDoc.m2

```diff
@@ -4363,6 +4372,66 @@ doc ///
         freeResolution
 ///
 
+doc ///
+    Key
+        (Ext,Module,Module)
+        (Ext,Ideal,Ideal)
+        (Ext,Ideal,Module)
+        (Ext,Ideal,Ring)
+        (Ext,Module,Ideal)
+        (Ext,Module,Ring)
+    Headline
+        total Ext module
+    Usage
+        Ext(M, N)
+    Inputs
+        M:Module
+            or ofClass{Ideal,Ring}, that is homogeneous
+        N:Module
+            or ofClass{Ideal,Ring} over the same ring as $M$, that is also homogeneous
```

Comment:
I don't think `ofClass{Ideal,Ring}` works like this ...
```suggestion
        M:{Ring,Ideal,Module}
            that is homogeneous
        N:{Ring,Ideal,Module}
            over the same ring as $M$, that is also homogeneous
```

Reply:
Thanks for reminding/teaching us about this syntax!